packages/contracts/contracts/core/HackMoneyVault.sol

```diff
@@ -23,6 +23,7 @@ contract HackMoneyVault is Multicall, Ownable, BaseVault {
     // Amount locked for scheduled withdrawals last week;
     uint128 public lastQueuedWithdrawAmount;
     // % of funds to be used for weekly option purchase
+    // TODO: Do we need this?
```

Comment:
We dont need it, unless we introduce a limit on number of amount of options sold and have this to be upgradable. Idea is to start the system slowly and lift limitation with time

Reply:
Right, but does not vault cap solve this?

Reply:
Yes you are right, since we have one strategy per vault